### PR TITLE
Feature/small back end tasks

### PR DESCRIPTION
**Pull request by simaob**
- Adds batch actions for Litigations and Targets (delete, archive, publish);
- Adds publish batch action to Laws;

### CODE REVIEW

File: app/admin/litigations.rb [hunk 1]
```diff
@@ -113,6 +112,46 @@
 
   form partial: 'form'
 
+  batch_action :publish,
```
Comment (tsubik) [inline app/admin/litigations.rb]: See a potential here to refactor those to some kind of DSL extensions that we have. But could be done in another PR.

File: app/commands/command/batch/publish.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    class Publish
+      def initialize(collection, ids)
+        @collection = collection
+        @ids = ids
+      end
+
+      def call
+        @collection
+          .where(id: [*@ids])
+          .update_all(visibility_status: 'published')
```
Comment (tsubik) [inline app/commands/command/batch/publish.rb]: `update_all` won't call any callbacks, and when we merge public_activity PR we either need to add one activity like "George published x Litigations" and put ids in the comment. Or fire `@collection.where(id: [*ids]).update(visibility_status: 'published')`.

Comment (tsubik) [reply app/commands/command/batch/publish.rb]: Which will generate bunch of activites. Not sure what is better, I think just one batch activity seems fine.

File: spec/factories/litigations.rb [hunk 3]
```diff
@@ -32,7 +32,7 @@
     citation_reference_number { 'SFKD777FDK77' }
     summary { 'Summary Lorem ipsum dolor dalej nie pamietam' }
     core_objective { 'Core objectives Lorem ipsumumum' }
-    visibility_status { Litigation::VISIBILITY.sample }
+    visibility_status { Litigation::VISIBILITY.first }
```
Comment (tsubik) [inline spec/factories/litigations.rb]: right, maybe better just pick 'published'

File: app/commands/command/batch/publish.rb (hunk 2 above)
Comment (simaob) [reply app/commands/command/batch/publish.rb]: yeah, I think one batch activity would work well, and keep it saner. I'll update it.

Comment (simaob) [reply app/commands/command/batch/publish.rb]: actually, let's leave it like this and then we address capturing these on the activities.

**Pull request merged**